modules/concept-docs/pages/data-model.adoc

```diff
@@ -23,7 +23,7 @@ Values can be placed in the beginning or end of a list, and can be accessed usin
 * *Queue* is a wrapper over a _list_ which offers FIFO (first-in-first-out) semantics, allowing it to be used as a lightweight job queue.
 * *Set* is a wrapper over a _list_ which provides the ability to handle unique values.
 
-These data structures are stored as JSON documents in Couchbase, and can therefore be accessed using N1QL, Full Text Search, and normal key-value operations.
+These data structures are stored as JSON documents in Couchbase, and can therefore be accessed using {sqlpp} (formerly N1QL), Full Text Search, and normal key-value operations.
```

Comment:
Fine to leave in - but note just replacing N1QL with Query may also be as effective in cases like these.